text/1589-rustc-bug-fix-procedure.md

```diff
@@ -98,29 +98,31 @@ What follows is a template for tracking issues.
 
 ---------------------------------------------------------------------------
 
-This is the **summary issue** for the `YOUR_LINT_NAME_HERE`
-future-compatibility warning and other related errors. The goal of
-this page is describe why this change was made and how you can fix
-code that is affected by it. It also provides a place to ask questions
-or register a complaint if you feel the change should not be made. For
-more information on the policy around future-compatibility warnings,
-see our [breaking change policy guidelines][guidelines].
-
-[guidelines]: LINK_TO_THIS_RFC
-
-#### What is the warning for?
-
-*Describe the conditions that trigger the warning and how they can be
-fixed. Also explain why the change was made.**
-
-#### When will this warning become a hard error?
-                    
-At the beginning of each 6-week release cycle, the Rust compiler team
-will review the set of outstanding future compatibility warnings and
-nominate some of them for **Final Comment Period**. Toward the end of
-the cycle, we will review any comments and make a final determination
-whether to convert the warning into a hard error or remove it
-entirely.
+#### What is this lint about
+
+*Describe here what kinds of Rust code will cause the lint to trigger.*
+
+*Motivational text or historical information is also of use, but the
+most important thing is to explain what kind of code the lint is
+detecting.*
+
+*You may wish to include a link to this RFC as part of explaining the
+lint in some manner, for example:*
+
+For more information on the policy around future-compatibility
+warnings, see our [breaking change policy guidelines][RFC 1589].
+
+[RFC 1589]: https://github.com/rust-lang/rfcs/blob/master/text/1589-rustc-bug-fix-procedure.md
+
+#### How to fix this warning/error
+
+*Explain here what the developer needs to do to address the warning.*
+
+#### Current status
+
+- [ ] PR ? introduces the `YOUR_LINT_NAME_HERE` lint as warn-by-default
+- [ ] PR ? makes the `YOUR_LINT_NAME_HERE` lint deny-by-default
+- [ ] PR ? makes the `YOUR_LINT_NAE_HERE` lint a hard error
 
```

Comment:
It would be nice to include:

#### Schedule

*Describe the estimated schedule in terms of Rust releases for converting the warning to deny and then ultimately to a hard error.*

Reply:
Has any summary issue for a future-incompatibility lint included such a schedule?

I don't mind preserving parts of the original RFC that were good ideas, nor do I mind codifying existing practice, but in my quick skim earlier, I don't recall any future-incompatibility specifying a schedule.

Reply:
Not to my knowledge, but I think it would be good to have schedules for the purpose of effective tracking and triage; standardization of such schedules as Niko notes below would also be helpful imo.

Reply:
Okay; I don't dispute the idea that we need to have a more upfront focus on the scheduling. If @nikomatsakis signs off on the text you drafted, then I'll add it to this PR.

But if its something that's going to want a lot of discussion/tweaking/bikeshedding, then I'd rather leave it to a different Pull Request.